index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title></title>
+  </head>
+  <body>
+    <div id='shopping_list'>
+      <img src="big1.png" style="padding-left: 0ch;"><br>
+      <h1>{{ header?header : 'Welcome' }}</h1><br>
+      <div class="add_item_cart">
```

Comment:
@janardhanakulakarni always use classname with `-` ex `add-item-cart` instead of `add_item_cart`

Reply:
As you told I have changed the class name, Please have a look at it